concurrent/src/main/java/io/airlift/concurrent/MoreFutures.java

```diff
@@ -241,6 +241,34 @@ public static Throwable unwrapCompletionException(Throwable throwable)
     }
 
     /**
+     * Returns the result of the input {@link Future}, which must have already completed.
+     * <p>
+     * Similar to Futures{@link Futures#getDone(Future)}, but does not throw checked exceptions.
+     */
+    public static <T> T getDone(Future<T> future)
```

Comment:
These need basic coverage tests.  Just a checks with a success, not done, and has exception futures.